README.md

```diff
@@ -166,6 +166,29 @@ def lambda_handler(event, context):
     }
 ```
 
+## Replacing the SSM client
+
+You can replace the SSM client for testing by setting the `SSMParameterStore.SESSION_FACTORY` and / or the `SSMParameterStore.CLIENT_FACTORY` class variables.
+
+For example, if you are using the `placebo` library to mock out `boto3` behaviour:
+
+```python
+import boto3
+import placebo
+
+from ssm_cache.cache import SSMParameterStore
+
+def session_factory():
+    session = boto3.Session()
+
+    pill = placebo.attach(session, data_path='/path/to/response_directory')
+    pill.playback()
+
+    return session
+
+SSMParameterStore.SESSION_FACTORY = session_factory
```

Comment:
Should we add at least one test where we actually do this? :)

Reply:
👍